packages/combobox/src/Combobox.ts

```diff
@@ -227,6 +227,12 @@ export class Combobox extends Textfield {
         if (!this.availableOptions[nextActiveIndex].disabled) {
             this.activeDescendant = this.availableOptions[nextActiveIndex];
         }
+        this.optionEls.forEach((el) =>
```

Comment:
As part of the necessary accessibility fixes, `aria-activedescendant` has been removed from menu, so combobox, must now manage its own `aria-activedescendant`.

Note: Combobox is still not and never has been accessible. That fix will be a separate PR, but this at least ensures our tests and API don't break.